packages/create-sitecore-jss/src/initializers/nextjs/index.ts

```diff
@@ -0,0 +1,60 @@
+import chalk from 'chalk';
+import path from 'path';
+import { prompt } from 'inquirer';
+import { prompts, NextjsAnswer, Prerender } from './prompts';
+import { Initializer } from '../../common/Initializer';
+import { transform } from '../../common/steps';
+import { isDevEnvironment } from '../../common/utils/helpers';
+import { removeDevDependencies } from './remove-dev-dependencies';
+import { NextjsArgs } from './args';
+import { FetchWith } from '../../common/prompts/base';
+
+export default class NextjsInitializer implements Initializer {
+  get isBase(): boolean {
+    return true;
+  }
+
+  async init(args: NextjsArgs) {
+    const defaults = args.yes
+      ? {
+          appName: 'sitecore-jss-nextjs',
+          fetchWith: FetchWith.GraphQL,
+          prerender: Prerender.SSG,
+          hostName: 'https://cm.jss.localhost',
+          appPrefix: true,
```

Comment:
Change to false (consistent with jss create default)
```suggestion
          appPrefix: false,
```